src/assets/scripts/client/aircraft/AircraftTypeDefinitionModel.js

```diff
@@ -165,4 +178,33 @@ export default class AircraftTypeDefinitionModel extends BaseModel {
         this.speed = null;
         this.capability = null;
     }
+
+    /**
+     * Build the string used for `#icaoWithWeightClass`
+     *
+     * @for AircraftTypeDefinitionModel
+     * @method _buildIcaoWithWeightClass
+     * @return {string}
+     * @private
+     */
+    _buildIcaoWithWeightClass() {
+        const HEAVY_LETTER = 'H';
+        const SUPER_LETTER = 'U';
+        let aircraftIcao = this.icao;
+
+        switch (this.weightclass) {
+            case HEAVY_LETTER:
+                aircraftIcao = `${HEAVY_LETTER}/${this.icao}`;
+
+                break;
+            case SUPER_LETTER:
+                aircraftIcao = `${SUPER_LETTER}/${this.icao}`;
```

Comment:
And for completeness (though not necessarily accuracy), let's also include an equipment suffix of L for all aircraft. Will be goofy for cessnas, but better to have it than not. This equipment suffix should be applied to aircraft type strings for all weightclasses.

`${HEAVY_LETTER}/${this.icao}/L`